python/cudf/cudf/_lib/parquet.pyx

```diff
@@ -87,3 +108,37 @@ cpdef read_parquet(filepath_or_buffer, columns=None, row_group=None,
         df.index.name = new_index_name
 
     return df
+
+cpdef write_parquet(
+        cols,
+        path=None,
+        compression=None):
+    """
+    Cython function to call into libcudf API, see `write_parquet`.
+
+    See Also
+    --------
+    cudf.io.parquet.write_parquet
+    """
+
+    # Setup writer options
+    cdef parquet_writer_options options = parquet_writer_options()
+    if compression is None:
+        options.compression = compression_type.none
+    elif compression == "snappy":
+        options.compression = compression_type.snappy
+    else:
+        raise ValueError("Unsupported `compression` type")
+
+    # Create writer
+    cdef string filepath = <string>str(path).encode()
+    cdef unique_ptr[parquet_writer] writer
+    with nogil:
+        writer = unique_ptr[parquet_writer](
+            new parquet_writer(filepath, options)
+        )
+
+    # Write data to output
+    cdef unique_ptr[cudf_table] c_in_table = make_table_from_columns(cols)
```

Comment:
I believe we have a new table accessor that should be used here instead of using this `make_table_from_columns`.

Reply:
I changed the signature to accept a Table rather than columns and then modified parquet.py to pass df.view() instead of columns.

Reply:
This is still expecting `cols` instead of getting the `table_view`. I'd also expect this API to use a `table_view` instead of `cudf_table`.

Reply:
Just made the changes. Thanks